src/presetsController.ts

```diff
@@ -1717,62 +1718,89 @@ export class PresetsController implements vscode.Disposable {
             return undefined;
         }
 
+        const clonedPresetsFile = lodash.cloneDeep(presetsFile);
+
         log.info(localize('expanding.presets.file', 'Expanding presets file {0}', presetsFile?.__path || ''));
 
         const expansionErrors: ExpansionErrorHandler = { errorList: [], tempErrorList: []};
 
-        const expandedConfigurePresets = (await Promise.all((presetsFile?.configurePresets || []).map(async configurePreset =>
-            preset.expandConfigurePreset(
+        const expandedConfigurePresets = (await Promise.all((clonedPresetsFile?.configurePresets || []).map(async configurePreset => {
+            const inheritedPreset = await preset.getConfigurePresetInherits(
+                this.folderPath,
+                configurePreset.name,
+                true,
+                true,
```

Comment:
Shouldn't we be passing `false` for `usePresetsPlusIncluded` here?

Reply:
yes, thanks